addons/hr/static/tests/web/m2x_avatar_user_tests.js

```diff
@@ -108,7 +104,7 @@ QUnit.module("M2XAvatarUser", ({ beforeEach }) => {
         });
         // Open card
         await click(document, ".o_m2o_avatar > img");
-        assert.verifySteps(["setTimeout of 250ms", "user read"]);
+        assert.verifySteps(["setTimeout of 250ms"]);
```

Comment:
The `user read` was there to ensure the `mockRPC` is not dead code. Since forging `mockRPC` in test is cheating, at least we should show the cheating is used. Without it, this can likely be dead code due to nothing really asserting this is relevant for test.